### DISCUSSION TITLE
Are you guys affiliated with https://en.bem.info/ and why the stark difference?

### DISCUSSION THREAD

**Discussion by woodysee** [getbem/getbem.github.io#238] [Q&A]
I was doing some reading up on implementing BEM for my SASS modules and have suddenly noticed that there is an [alternative BEM standard](https://github.com/bem-site/bem-method) named ["bem.info"](https://en.bem.info/methodology/css/) that contradicts the standard set by "Get BEM".

Both standards are also different. For example, "Get BEM" requires modifiers to be separated from blocks and elements with double dashes (i.e.`naming__cell--modifier--small`) while "bem.info" requires modifiers to [be separated using a single underscore](https://en.bem.info/methodology/css/#modifiers) (i.e. `naming__cell modifier_small`).

I have first learnt about BEM from this site. So the emergence of a different and competing standard has presented a dilemma on which source of truth for the BEM methodology I should follow.

Both "Get BEM" and ["bem.info"](https://en.bem.info/) has stated that they at least both done work with Yandex at some point. They even both have the same logo.

So I am wondering if there is any affiliation between "Get BEM" and "bem.info"? And why are two different standards being set on both sites? Which standard of BEM I should use?

**Comment by maybephilipp** [comment]
I just found this issue: bem-site/bem-method#775
I think we can treat this style guide like alternative naming convention.
Also `bem.info` is more developed method. There are also conventions for Javascript and other areas, libraries and tools.
As I see, `getbem` has only CSS naming convention.

PS. I like `getbem` more, it seems more convenient to me.

**Comment by maybephilipp** [comment]
Also: https://github.com/getbem/getbem.com/issues/8

There are description and discussion why `getbem` devs decided to create their own website. And seems I agree with them.
